core/src/main/java/org/testcontainers/containers/GenericContainer.java

```diff
@@ -867,7 +868,6 @@ public void setWaitStrategy(org.testcontainers.containers.wait.strategy.WaitStra
      * @see #waitingFor(org.testcontainers.containers.wait.strategy.WaitStrategy)
      */
     protected void waitUntilContainerStarted() {
-        org.testcontainers.containers.wait.strategy.WaitStrategy waitStrategy = getWaitStrategy();
```

Comment:
No sure if it was made on purpose. The local variable hides the instance variable and seemed redundant.